scripts/wrappers/remove_node.py

```diff
@@ -25,7 +25,13 @@
 def remove_dqlite_node(node, force=False):
     try:
         # If node is an IP address, find the node name.
-        if type(ip_address(node)) is IPv4Address:
+        is_node_ip = False
+        try:
+            is_node_ip = True if ip_address(node) is IPv4Address else False
+        except ValueError:
+            pass
```

Comment:
Perhaps this is an easier way to write the same?

```suggestion
        is_node_ip = True
        try:
            ip_address(node)
        except ValueError:
            is_node_ip = False
```

Reply:
Oh, I missed the `type()` check. Thanks for the review!